.eslintrc.js

```diff
@@ -29,6 +29,8 @@ module.exports = {
     'react/jsx-uses-vars': 2,
     '@typescript-eslint/no-use-before-define': 0,
     '@typescript-eslint/no-var-requires': 0,
-    '@typescript-eslint/explicit-function-return-type': 0
+    '@typescript-eslint/explicit-function-return-type': 0,
+    '@typescript-eslint/ban-types': 0,
+    '@typescript-eslint/ban-ts-comment': 1
```

Comment:
Why are you adding these ruls? Are they part of your fix?

Reply:
The existing eslint config extends the "recommended" list from both eslint and typescript-eslint:
https://github.com/wix/react-native-calendars/pull/2056/commits/4301d130b753ece6919e712bdfae68721187db5b#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5L12
In the new version of typescript-eslint, some new default rules are added.

I mentioned this in the commit message, but forgot to also include it in the pull request description (added that now):
> I disabled some of the new rules that caused a lot of errors, but they can probably be fixed and enabled in subsequent commits (there are a lot of other lint warnings that can be tackled as well, though).

Reply:
Thanks for clarifying!